apps/sms/test/marionette/lib/views/thread_list.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+var View = require('./view');
+
+function ThreadList() {
+  View.apply(this, arguments);
+}
+module.exports = ThreadList;
+
+ThreadList.prototype = {
+  __proto__: View.prototype,
+
+  goToFirstThread: function() {
+    this.pageAccessors.ThreadList.firstThread.tap();
+    var Thread = require('./thread');
```

Comment:
Rebased. As the rest of the comment is linked to the initialization of subpages, let's move this discussion on general thread.